client/src/components/Instructions/index.js

```diff
@@ -92,6 +92,100 @@ export default class Instructions extends Component {
     );
   }
 
+  renderGaslessCounterSetup() {
+    const addressDefault = this.props.ganacheAccounts.length > 2 ? this.props.ganacheAccounts[2] : '<ADDRESS>';
+    return (
+      <div className={styles.instructions}>
+        <h2> Build your first app with ZepKit </h2>
+        <div className={styles.step}>
+          <div className={styles.instruction}>
+            1. Ensure Docker is installed on your computer.
+          </div>
+          <div className={styles.code}>
+            <code>
+              <a href="https://docs.docker.com/install/">https://docs.docker.com/install/</a>
+            </code>
+          </div>
+        </div>
+        <div className={styles.step}>
+          <div className={styles.instruction}>
+            2. In another terminal, run the gas relay docker container with ganache, wait a few seconds for it to sping up (run inside your zepkit folder).
+          </div>
+          <div className={styles.code}>
+            <code>
+              npm explore tabookey-gasless npm run gsn-dock-relay-ganache
+            </code>
+          </div>
+        </div>
+        <div className={styles.step}>
+          <div className={styles.instruction}>
+            3. In a different terminal, init your project with ZeppelinOS (run inside your zepkit folder).
+          </div>
+          <div className={styles.code}>
+            <code>
+              zos init zepkit
+            </code>
+          </div>
+        </div>
+        <div className={styles.step}>
+          <div className={styles.instruction}>
+            4. Add the GaslessCounter contract to your project (Note that a compilation warning will be thrown, you can ignore this).
+          </div>
+          <div className={styles.code}>
+            <code>
+              zos add GaslessCounter
+            </code>
+          </div>
+        </div>
+        <div className={styles.step}>
+          <div className={styles.instruction}>
+            5. Connect with your local blockchain by opening a session (Note that we automatically prefilled your ganache account).
+          </div>
+          <div className={styles.code}>
+            <code>
+              zos session --network development --from {addressDefault} --expires 3600
+            </code>
+          </div>
+        </div>
+        <div className={styles.step}>
+          <div className={styles.instruction}>
+            6. Let's compile the Gasless Counter contract.
```

Comment:
This step is actually deploying the logic contract, not compiling it

Reply:
I copied the verbage from the Counter tutorial for this step, since it is deploying the contract, should I change this on the other tutorial as well? Fixing in next commit for GaslessCounter